sensu/plugins/check-large-receive-offload.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+#
+# Checks that large receive offload is disabled
+#
+# Return CRITICAL if large receive offload is enabled
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+
+import subprocess
+import argparse
+import sys
+import re
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+CRITICALITY = 'critical'
+
+def switch_on_criticality():
+    if CRITICALITY == 'warning':
+        sys.exit(STATE_WARNING)
+    else:
+        sys.exit(STATE_CRITICAL)
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-d', '--devices', help='provide eth devices for bond0')
+    parser.add_argument('-z', '--criticality', default='critical')
+    args = parser.parse_args()
+
+    CRITICALITY = args.criticality
+
+    if args.devices is None:
+        parser.print_help()
+        switch_on_criticality()
+
+    args.devices = re.findall('eth[0-7]', args.devices)
```

Comment:
should we just split this by `,` and expect the user to provide a comma seperated list in args ?